quantum/audio/musical_notes.h

```diff
@@ -61,7 +61,7 @@
 
 // Notes - # = Octave
 
-#define NOTE_REST         0.00f
+#define NOTE_REST         1.00f
```

Comment:
This creates a weird behavior on ARM where it completely stops playback. 

```suggestion
#ifdef __arm__
#define NOTE_REST         1.00f
#else
#define NOTE_REST         0.00f
#endif
```

Reply:
I've already done this, and can verify it works.
https://github.com/qmk/qmk_firmware/blob/4cde82ef57ff2f00dc637ac41175ace2f551da47/users/drashna/config.h#L18-L21